.github/workflows/release.yaml

```diff
@@ -83,4 +84,31 @@ jobs:
         env:
           # Zone is required by both authentication methods
           CLOUDFLARE_ZONE: ${{ secrets.CLOUDFLARE_ZONE_ID }}
-          CLOUDFLARE_TOKEN: ${{ secrets.CLOUDFLARE_TOKEN }}
\ No newline at end of file
+          CLOUDFLARE_TOKEN: ${{ secrets.CLOUDFLARE_TOKEN }}
+
+      - name: Create GitHub Release
+        id: create_release
```

Comment:
Where did this new code come from? Why do we need it?

Reply:
This is same as `release.yaml` in master branch